tests/phpunit/CommonExtractorTest.php

```diff
@@ -286,7 +288,7 @@ public function testRetries(): void
 
         try {
             ($this->getApp($config))->run();
-        } catch (UserException $e) {
+        } catch (UserExceptionInterface $e) {
```

Comment:
Pouzivaju sa tu exceptions z noveho balika `db-extractor-adapter`, preto je to tu nahradene s interface.